ui/src/components/htmlhintcomponents/UpdateHTMLRules.svelte

```diff
@@ -0,0 +1,104 @@
+<script>
+  import Toastr from "../misccomponents/Toastr.svelte";
+  import { CONSTS } from "../../utils/utils";
+  import Modal from "../misccomponents//Modal.svelte";
+  import LoadingFlat from "../misccomponents/LoadingFlat.svelte";
+  import slug from "slug";
+import { keys, last } from "ramda";
+  
+  export let url;
+  export let show;
+  export let loading;
+  export let user;
+
+  let saving;
+  let addedSuccess;
+
+	let selection = [];
+
+  const dismiss = () => (show = false);
+
+  const updateIgnore = async () => {
+    const selectedRules = selection.toString()
+    saving = true;
+    if (selection.length > 0) {
+      const res = await fetch(
+        `${CONSTS.API}/api/config/${user.apiKey}/htmlhintrules`,
+        {
+          method: "PUT",
+          body: JSON.stringify({
+            url,
+            selectedRules,
+          }),
+          headers: { "Content-Type": "application/json" },
+        }
+      );
+  
+      if (res.ok) {
+        saving = false;
+        show = false;
+        addedSuccess = true;
+      } else {
+        throw new Error("Failed to load");
+      }
+    } else {
+        alert('select something')
+    }
+  };
+  
+  const htmlHintRules = [
+     { rule: "tagname-lowercase" },
+     { rule: "attr-lowercase" }, 
+     { rule: "attr-value-double-quotes" },
+     { rule: "attr-value-not-empty" },
+     { rule: "attr-no-duplication" },
+     { rule: "doctype-first" },
+     { rule: "tag-pair" },
+     { rule: "empty-tag-not-self-closed" },
+     { rule: "spec-char-escape" },
+     { rule: "id-unique" },
+     { rule: "src-not-empty" },
+     { rule: "title-require" },
+     { rule: "alt-require" },
+     { rule: "doctype-html5" },
+     { rule: "style-disabled" },
+     { rule: "inline-style-disabled" },
+     { rule: "inline-script-disabled" },
+     { rule: "id-class-ad-disabled" },
+     { rule: "href-abs-or-rel" },
+     { rule: "attr-unsafe-chars" },
+     { rule: "head-script-disabled" },
```

Comment:
What happens when HtmlHint add new rules?
We should load all rule names from HtmlHint instead of hard coding them here.

We might need to do it this way for custom rules though.